app/views/course/assessment/submissions/_submission.html.slim

```diff
@@ -0,0 +1,18 @@
+- assessment = submission.assessment
+= content_tag_for(:tr, submission) do
+  td
+    = link_to_course_user(submission.course_user) do
+      = format_inline_text(submission.course_user.name)
```

Comment:
Do we need this block? 

Seems like `link_to_course_user` basically does this for you, except that it does not pass the user's name through `format_inline_text`. Perhaps we should add `format_inline_text` to `Course::ControllerHelper#display_course_user` instead?